pkg/makefile

```diff
@@ -161,10 +161,10 @@ deb: stage_deb
 		-s dir \
 		-d nfs-kernel-server \
 		-d net-tools \
-		-d libdevmapper \
-		-d libdevmapper-event \
+		-d libdevmapper1.02.1 \
+		-d libdevmapper-event1.02.1 \
 		-d nfs-common \
-		-d 'lxc-docker-1.7.1' \
+		-d 'docker-engine (= 1.7.1-0~trusty)' \
```

Comment:
I have not for the life of me been able to get this to install with anything other than the complete version description as above. This sucks for installing to multiple versions of Ubuntu, of course. If anyone has any ideas, I'm all ears.